docs/blog/newsletter-december-2019.rst

```diff
@@ -0,0 +1,127 @@
+
+So that's all on the news front. We’ve had another month of great conversations and insights from the Write the Docs Slack - enjoy!
+
+----------------------------------------------------
+Documentation feedback widgets - thumbs up or down?
+----------------------------------------------------
+
+Getting user feedback on your documentation can be challenging (and not just emotionally!). This month, our community discussed options for asking for that feedback on our help sites.
+
+While most felt that getting direct user feedback was ultimately more useful, often that's not possible. Implementing a widget that asks users for star ratings or a thumbs up/thumbs down can get you somewhere - but these ratings systems have their challenges.
+
+One community member suggested that readers may only rate docs when they're unhappy with their experience, skewing the results to the negative. Others noted that while these can give an idea of high-level likes and dislikes, a thumbs down alone doesn't tell you what in a doc needs improving or fixing. One idea was to have a widget where, if you give a thumbs down, you're prompted to leave a comment. But increasing the friction for providing feedback decreases the amount of feedback you'll receive.
+
+There was another theory for the prevalence of negative comments. Most documentation is written intentionally without the "personal touch" , or without even a byline, that would let our audiences connect with the person behind the writing.
+
+Here is some recommended reading:
+
+* `Maybe Our Documentation "Best Practices" Aren't Really Best Practices <https://kayce.basqu.es/blog/best-practices#todos>`__
+* `To Allow Blog Comments or Not? Here's What the Data Shows... <https://optinmonster.com/to-allow-blog-comments-or-not-heres-what-the-data-shows/>`__
+
+As well as some tool suggestions:
+
+* `Delighted <https://delighted.com/>`_
+* `Usabilla <https://usabilla.com/>`_
+* `Form Keep <https://formkeep.com/>`_
+* `User Voice <https://www.uservoice.com/>`_
+* `Satismeter <https://www.satismeter.com/>`_
+
+If you want to chat more about feedback widget options, jump into #doc-tools!
+
+-------------------------------
+Our favorite keyboard shortcuts
+-------------------------------
+
+Nothing makes you more nostalgic for familiar keyboard shortcuts than switching operating systems! Fortunately, many Windows keyboard shortcuts work for Mac if you use Command instead of Ctrl (and vice versa). But getting comfortable takes time. To help you settle into a new OS - or just learn some handy tricks - here's a roundup of the community's favorite shortcuts.
+
+* Mac:
+   * Command + Backquote: switch between windows in one app (eg different browser windows).
```

Comment:
maybe some kind of emphasis for the combos? code or bold or whatever

Reply:
good idea, done